packages/react-native/src/components/Fabric/Video.tsx

```diff
@@ -0,0 +1,31 @@
+import 'react-native-get-random-values';
+import React, { useState } from 'react';
+import type * as SignalWire from '@signalwire/js';
+import { Fabric } from '@signalwire-community/react';
+
+import { registerGlobals } from 'react-native-webrtc';
+import RemoteStream from '../RemoteStream';
+registerGlobals();
+
+interface IVideoNativeProps extends Fabric.IVideoProps {
+  style: any;
+}
+
+export function Video({ style, onRoomReady, ...props }: IVideoNativeProps) {
+  const [roomSession, setRoomSession] =
+    useState<SignalWire.Video.RoomSession | null>(null);
+
+  return (
+    <Fabric.CoreVideo
+      {...props}
+      onRoomReady={(r: SignalWire.Video.RoomSession) => {
+        setRoomSession(r);
+        onRoomReady?.(r);
+      }}
+      audio={props.audio ?? true}
+      video={props.video ?? true}
+    >
+      <RemoteStream roomSession={roomSession ?? undefined} style={style} />
```

Comment:
note: this component will need to be updated to accept either a roomSession or whatever type of objects the Fabric namespace will produce. Let's keep it like this for now.